stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/view/MediaAttachmentsGroupView.kt

```diff
@@ -178,6 +178,14 @@ internal class MediaAttachmentsGroupView : ConstraintLayout {
             .getOrDefault(0f)
     }
 
+    private fun createMediaAttachmentView(): MediaAttachmentView {
+        return MediaAttachmentView(context).also {
```

Comment:
Don't we use `apply` for these situations?

Reply:
When I did `attachmentClickListener = attachmentClickListener` I was just assigning it to itself, any other NIT solution?

Reply:
Semantically, `apply` would be a much better fit here indeed. However, the name conflicts make it kinda awkward. Maybe it's still worth going with, `it.x = x` might as well be `this.x = x`.

Reply:
`this.x = x` not working as we expect

Reply:
Oh right, that would be broken too 🤦‍♂️  Right, so `also` is the way to go here 😄

Reply:
or `attachmentClickListener = this@MediaAttachmentsGroupView.attachmentClickListener`

Reply:
`apply` is probably too error prone in this case to leave it in. If someone comes by and adds another item later on, they can forget to add the qualifier, etc.

Reply:
Ok, used also